src/components/navigation/LogoLink.tsx

```diff
@@ -0,0 +1,6 @@
+export default function LogoLink(props: {
+  href: string;
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children?: ReactNode;
```

and also

`import type { ReactNode } from 'react';`